pkg/expr/functions/cairo/png/cairo.go

```diff
@@ -1532,13 +1543,13 @@ func setupTwoYAxes(cr *cairoSurfaceContext, params *Params) {
 	params.yTopR = params.yStepR * math.Ceil(yMaxValueR/params.yStepR)
 
 	if params.logBase != 0 {
-		if yMinValueL > 0 && yMinValueR > 0 {
+		if yMinValueL > 0 && yMaxValueL > 0 {
```

Comment:
It seems this change isn't described in commit msg. Is it intentional?

Reply:
No, it's a typo. 🤦 Fixed, thanks!